src/steps/service/StepsService.ts

```diff
@@ -1,11 +1,14 @@
 import { AxiosResponse } from 'axios';
 import * as Q from 'q';
+import ResponseStatus from '../../model/ResponseStatus';
 
 import Resource from '../../model/Resource';
 import Version from '../../model/Version';
 import RequestClientWrapper from '../../RequestClientWrapper';
 import GenerateApiError from '../../utils/GenerateApiError';
 import Step from '../model/Step';
+import StepCreateRequest from '../model/StepCreateRequest';
+import StepUpdateRequest from '../model/StepUpdateRequest';
 
 const EXPECTED_GET_STATUS = 200;
```

Comment:
Remove this if it's no longer needed here. It's scoped to this service so it should be fairly easy to update this.